code/datums/elements/muffles_speech.dm

```diff
@@ -0,0 +1,47 @@
+/datum/element/muffles_speech
+
+/datum/element/muffles_speech/Attach(datum/target)
+	. = ..()
+	if(!isitem(target))
+		return ELEMENT_INCOMPATIBLE
+
+	RegisterSignal(target, COMSIG_ITEM_EQUIPPED, PROC_REF(equipped))
+	RegisterSignal(target, COMSIG_ITEM_DROPPED, PROC_REF(dropped))
+
+/datum/element/muffles_speech/Detach(datum/source)
+	. = ..()
+	UnregisterSignal(source, list(COMSIG_ITEM_EQUIPPED, COMSIG_ITEM_DROPPED))
+
+/datum/element/muffles_speech/proc/equipped(obj/item/source, mob/user, slot)
+	SIGNAL_HANDLER
+	if(source.slot_flags & slot)
+		RegisterSignal(user, COMSIG_MOB_SAY, PROC_REF(muzzle_talk))
+		RegisterSignal(user, COMSIG_MOB_PRE_EMOTED, PROC_REF(emote_override))
+
+/datum/element/muffles_speech/proc/dropped(obj/item/source, mob/user)
+	SIGNAL_HANDLER
+	UnregisterSignal(user, list(COMSIG_MOB_PRE_EMOTED, COMSIG_MOB_SAY))
+
+/datum/element/muffles_speech/proc/emote_override(datum/source, key, params, type_override, intentional, datum/emote/emote)
+	SIGNAL_HANDLER
+	if(!emote.hands_use_check && emote.emote_type & EMOTE_AUDIBLE)
+		return "makes a [pick("strong ", "weak ", "")]noise."
+	return COMPONENT_CANT_EMOTE
+
```

Comment:
You can't just randomly return a string from a signal and expect it to work, you have to insert a custom message on your own

```suggestion
/datum/element/muffles_speech/proc/emote_override(mob/living/source, key, params, type_override, intentional, datum/emote/emote)
	SIGNAL_HANDLER
	if(!emote.hands_use_check && (emote.emote_type & EMOTE_AUDIBLE))
		source.audible_message("makes a [pick("strong ", "weak ", "")]noise.", audible_message_flags = EMOTE_MESSAGE|ALWAYS_SHOW_SELF_MESSAGE)
		return COMPONENT_CANT_EMOTE
	return NONE
```

Also pls test ur code